libs/cimit-service/src/test/java/uk/gov/di/ipv/core/library/cimit/CiMitServiceTest.java

```diff
@@ -483,6 +484,49 @@ void getContraIndicatorsVCJwtWhenInvalidJWT() throws JsonProcessingException {
                                 TEST_USER_ID, GOVUK_SIGNIN_JOURNEY_ID, CLIENT_SOURCE_IP));
     }
 
+    @Test
+    void getContraIndicatorsVCJwtWhenInvalidCimitKey() throws JsonProcessingException {
```

Comment:
Rename to `getContraIndicatorsVCJwtThrowsErrorWhenInvalidCimitKey`

Reply:
done